modules/serverless-installing-cli-linux-ibm-z_rpm.adoc

```diff
@@ -0,0 +1,49 @@
+[id="installing-cli-linux_ibm_z_rpm_{context}"]
+
+== RPM Installation
+
+For Red Hat Enterprise Linux (RHEL), you can install `kn` as an RPM if you have an active {product-title} subscription on your Red Hat account.
+
+
+.Procedure
+. Enter the command:
```

Comment:
Instead of saying enter the command every time, maybe make these steps more descriptive, e,g, "Register the Subscription Manager:"
Not sure if that's accurate so don't quote me but just as an example - tell the user what the steps actually do.

Reply:
Also, same comment, remove all the unnecessary spacing - goes for the whole PR
Example
```
. Enter the command:
+
[source,terminal]
----
# subscription-manager refresh
----
```